orte/mca/plm/base/plm_base_launch_support.c

```diff
@@ -700,6 +701,9 @@ void orte_plm_base_post_launch(int fd, short args, void *cbdata)
                              "%s plm:base:launch job %s is not a dynamic spawn",
                              ORTE_NAME_PRINT(ORTE_PROC_MY_NAME),
                              ORTE_JOBID_PRINT(jdata->jobid)));
+        if (ORTE_PROC_IS_HNP) {
+            orte_submit_job_state_update(jdata, ORTE_JOB_STATE_RUNNING);
```

Comment:
The return code here is ignored, should anything be done (or make it a void)?

Reply:
Nothing to be done here. I'd keep it `int` for now just in case we need to do something in the future.